Kernel/Graphics/GraphicsManagement.cpp

```diff
@@ -56,7 +56,74 @@ void GraphicsManagement::activate_graphical_mode()
 
 static inline bool is_vga_compatible_pci_device(PCI::Address address)
 {
-    return PCI::get_class(address) == 0x3 && PCI::get_subclass(address) == 0x0;
+    // Note: Check for Display Controller, VGA Compatible Controller or
+    // Unclassified, VGA-Compatible Unclassified Device
+    return (PCI::get_class(address) == 0x3 && PCI::get_subclass(address) == 0x0) || (PCI::get_class(address) == 0x0 && PCI::get_subclass(address) == 0x1);
```

Comment:
Perhaps this would be a bit more readable:
```c++
    auto is_vga_compatible = PCI::get_class(address) == 0x3 && PCI::get_subclass(address) == 0x0;
    auto is_other_kind_of_vga_compatible_idk = PCI::get_class(address) == 0x0 && PCI::get_subclass(address) == 0x1;
    return is_vga_compatible || is_other_kind_of_vga_compatible_idk;
```
Would at least save people wondering what "class 3 subclass 0" or "class 0 subclass 1" is - I, for one, don't know that :P

Reply:
I'd so. I'd also recommend to read this - https://wiki.osdev.org/PCI#Class_Codes to better understand the numbers.